feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go

```diff
@@ -82,6 +82,7 @@ const (
 	asPathRepeatValue      = 3
 	aclStatement1          = "10"
 	aclStatement2          = "20"
+	NetworkInstance        = "DEFAULT"
```

Comment:
The network instance name seems to be vendor specific. Arista uses lower case "default". We can parameterize the vendor specific behavior by adding a deviation flag:

https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/deviations.go

```
var (
  DefaultNetworkInstance = flag.String("deviation_default_network_instance", "DEFAULT", "The name used for the default network instance for VRF.")
)
```

Would you mind also do a find and replace so that all occurrences of the default network instance name uses the value from the deviation flag consistently across the device tests?

(These are tests under ate_tests, otg_tests, or tests, but ignore the config gen tests.)

Reply:
Hi Likai,

Thanks for the reviewing the changes.
I have updated deviations.go file . So in my script should i use variable "DefaultNetworkInstance " ? and initialize the value to "DEFAULT" as well? please could you clarify.

Reply:
Yes, in your code, use *deviations.DefaultNetworkInstance, and initialize the value to DEFAULT (see my code snippet above).

Reply:
Got it, Thanks Likai. 
I have updated the script with *deviations.DefaultNetworkInstance and validated it.
After this PR, will work on other test.go files.